<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>가상화폐 상승장/하락장 알리미</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../image/iconfinder_cryptocurrency_blockchain_Bitcoin_BTC_2907507.png</normaloff>../image/iconfinder_cryptocurrency_blockchain_Bitcoin_BTC_2907507.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>741</width>
      <height>331</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>가상화폐</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>현재가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5일 이동평균</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>상승장/하락장</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>소요시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>업데이트시점</string>
     </property>
    </column>
    <column/>
   </widget>
   <widget class="QPushButton" name="pushButton_Initialize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last updated: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TimeStamp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>선물 잔고</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BalanceFuture">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_BalanceFuture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>현물 잔고</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BalanceSpot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_BalanceSpot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>총 잔고</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BalanceTotal">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_BalanceTotal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bithumb API 호출 횟수</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Binance API 호출 횟수</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_api_cnt_bithumb">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_api_cnt_bithumb</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_api_cnt_binance">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_api_cnt_binance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dbGenerator">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>630</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DB Generation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dbgeneration_finish_yn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
